### PR TITLE
Patch 5: Updates Website

### PR DESCRIPTION
**Pull request by xItzDiamondx**
**Resolves**

Resolves none

**Changes**

I made the website a bit better
**Reason for changes**

**Tests**
I have tested on my fork

**Extra Context**
If going to merge, please squash and merge,

### CODE REVIEW

**Comment by FunctionalMetatable** [comment]
please make new branches for pr'ing. the commit history is very long

**Comment by gosoccerboy5** [comment]
> please make new branches for pr'ing. the commit history is very long

Agreed. In addition, develop locally with [Github Desktop](//desktop.github.com) if possible, and make sure to squash and merge when merging PR with many commits

**Comment by xItzDiamondx** [comment]
I had GHD but I deleted it because my PC was running too slowly. I do use
VS Code and test locally though.

File: contributors.html [hunk 1]
```diff
@@ -30,6 +30,9 @@ <h4>Scratch Bookmarklets</h4>
   <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
+  <br><br><br>
+  
+  <div align=center>
```
Comment (Shluffy) [inline contributors.html:35]: Pls change this to above the `ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section `

**Review by Shluffy — CHANGES_REQUESTED** [review]

File: contributors.html [hunk 2]
```diff
@@ -41,6 +44,7 @@ <h4>Scratch Bookmarklets</h4>
     <td align="center"><a href="https://zi-youtuber.github.io/My-Website-1/"><img src="https://avatars.githubusercontent.com/u/82158378?v=4?s=100" width="100px;" alt=""/><br /><sub><b>ZI Coder</b></sub></a><br /><a href="https://github.com/Scratch-Bookmarklets/Scratch-Bookmarklets.github.io/commits?author=ZI-Youtuber" title="Code">💻</a> <a href="https://github.com/Scratch-Bookmarklets/Scratch-Bookmarklets.github.io/commits?author=ZI-Youtuber" title="Tests">⚠️</a> <a href="https://github.com/Scratch-Bookmarklets/Scratch-Bookmarklets.github.io/issues?q=author%3AZI-Youtuber" title="Bug reports">🐛</a> <a href="https://github.com/Scratch-Bookmarklets/Scratch-Bookmarklets.github.io/commits?author=ZI-Youtuber" title="Documentation">📖</a></td>
   </tr>
 </table>
+  </div>
```
Comment (Shluffy) [inline contributors.html:47]: again, this should be outside of the all-contributors stuff

**Review by Shluffy — CHANGES_REQUESTED** [review]

**Comment by Shluffy** [comment]
otherwise, everything looks good.

**Comment by Shluffy** [comment]
This only solves the problem on the contributors page, not on all of them.

**Comment by Shluffy** [comment]
the purpose of this pr has already been fixed.

**Comment by xItzDiamondx** [comment]
it has a few more patches so ima merge

**Review by xItzDiamondx — COMMENTED** [review]
Fixed

**Review by Shluffy — APPROVED** [review]
looks good. there doesn't seem to be any merge conflicts so I will merge.

**Pull request merged**